charts/oam-kubernetes-runtime/templates/oam-controller.yaml

```diff
@@ -39,6 +39,13 @@ rules:
   - apps
   resources:
   - deployments
+  - controllerrevisions
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
```

Comment:
Is this still needed when webhook is not deployed? Should we make it conditional on that?

Reply:
yes, i think we will still need it for Definition -> CRD Reference logic

Reply:
> i think we will still need it for Definition -> CRD Reference logic even the webhook not open.

Are you sure? I can't see any evidence of that: https://github.com/crossplane/oam-kubernetes-runtime/search?l=Go&q=CustomResourceDefinition

Reply:
I'm thinking of this issue https://github.com/crossplane/oam-kubernetes-runtime/issues/202 @negz